src/OutputWriters/windowed_time_average.jl

```diff
@@ -103,8 +103,8 @@ Base.copy(sch::AveragedTimeInterval) = AveragedTimeInterval(sch.interval, window
 
 A schedule for averaging over windows that precede SpecifiedTimes.
 """
-mutable struct AveragedSpecifiedTimes{W} <: AbstractSchedule
-    specified_times :: SpecifiedTimes
+mutable struct AveragedSpecifiedTimes{S<:SpecifiedTimes, W} <: AbstractSchedule
```

Comment:
```suggestion
mutable struct AveragedSpecifiedTimes{S, W} <: AbstractSchedule
```

not that its bad, but we don't use this style right now in Oceananigans. Basically we want type restrictions to have a specific purpose